core/src/MegaMolGraph.cpp

```diff
@@ -633,6 +633,14 @@ bool megamol::core::MegaMolGraph::add_call(CallInstantiationRequest_t const& req
         return false;
     }
 
+#ifdef PROFILING
+    std::vector<std::string> callbacks(call_description->FunctionCount());
+    for (uint32_t x = 0; x < call_description->FunctionCount(); ++x) {
+        callbacks[x] = call_description->FunctionName(x);
+    }
+    call->setProfilingInfo(callbacks);
+#endif
```

Comment:
I think we could move this to `CallAutoDescription<T>::CreateCall()` or decide to give each call all its function names regardless whether `PROFILING` is active or not?